tokenizer.go

```diff
@@ -3,7 +3,7 @@ package tokenizers
 // TODO packaging: how do we build the rust lib for distribution?
 
 /*
-#cgo LDFLAGS: ${SRCDIR}/libtokenizers.a -ldl -lm -lstdc++
+#cgo LDFLAGS: -ltokenizers -ldl -lm -lstdc++
```

Comment:
all of this, just for this one line change

Reply:
let's fix `make test`

Reply:
Done.  Also make the releases dependent on the test target so tests are ran during the make release flow.